migrations/slick/src/main/scala/Commands.scala

```diff
@@ -197,15 +197,8 @@ object M${version} {
   ))
 }
 """
-      case DBIO =>
-        val imports =
-          if (version > 1)
-            s"""import datamodel.v${version - 1}.schema.tables.Users
-import datamodel.v${version - 1}.schema.tables.UsersRow"""
-          else ""
-        s"""import ${driverName}.api._
+      case DBIO => s"""import ${driverName}.api._
```

Comment:
I don't think this would be a good idea, because I think in a Slick migration, the code usually relies on some object of the former version (in this case `Users` and `UserRow`). Even though users can simply add these imports manually, having these imports in the template can save users quite some trouble, and remind users to always import the predecessor version of the generated code instead of the latest version or one of the other former versions. Therefore, I would like to keep these imports in the template.

However, it is indeed not appropriate to have `Users` and `UsersRow` in it. What do you think if we use the wildcard import style (i.e. `import datamodel.v${version - 1}.schema.tables._`)?